railties/lib/rails/generators/rails/app/templates/Gemfile

```diff
@@ -50,8 +50,5 @@ group :development do
 <% end -%>
 end
 <% end -%>
-<% if RUBY_PLATFORM.match(/bccwin|cygwin|emx|mingw|mswin|wince|java/) -%>
-
 # Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

Comment:
We should keep this one. `if Bundler::WINDOWNS` will not work correctly because the Gemfile.lock will be different between OS. So lets only remove the `if RUBY_PLATFORM`